torch/csrc/jit/passes/quantization.cpp

```diff
@@ -694,6 +694,29 @@ bool isWeightOfConvOrLinear(Value* v) {
       v,
       AtenFuncArgs({{"conv2d", 1}, {"linear", 1}}),
       CallFuncArgs({{"linear", 2}}));
+
+  return result;
+}
+
+bool checkValueIsWeight(Module& module, Graph* graph, Value* v) {
```

Comment:
can we have a better name? maybe just `isWeightOfConvOrLinear` and a comment that this is going through CallMethod

Reply:
or just `isWeight`